widgets/src/main/java/org/pentaho/gwt/widgets/client/listbox/CustomListBox.java

```diff
@@ -1081,7 +1126,7 @@ public void setCommand( Command command ) {
   // ====================================== Listener Implementations =========================== //
 
   public void itemSelected( ListItem listItem, Event event ) {
-    fPanel.setFocus( true );
+    setFocus( true );
```

Comment:
AFAIK, this is called internally only (albeit public) when the user selects an item from the list (dropdown, in the case of visible = 1).

With the above change, when editable, `setFocus` will place focus on `editableTextBox`.

When `editable`, consider adding here the just removed `editableTextBox.selectAll();` (from `updateSelectedDropWidget`). It would have to be added after the below ifs so that the textbox text has been updated already.

Similarly, to match old behavior completely, the same could be done on the KEY UP/DOWN handler code.